### PR TITLE
Fix formatting for all dockerfile queries

### LINKED ISSUE #1816: Fix formatting for all dockerfile queries
### Platform
*Docker*

### Description
*Run `opa fmt --write` for all dockerfile rego query files*

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
Signed-off-by: Rogério Peixoto <rogerio.peixoto@checkmarx.com>

Closes #1816

### Platform
*Docker*

### Description
*Run `opa fmt --write` for all dockerfile rego query files*

```
#!/usr/bin/env bash
shopt -s extglob
COUNT=0
for rego in echo ./assets/queries/dockerfile/**/*.rego; do
  echo $rego
  opa fmt -w $rego
  COUNT=$(( COUNT + 1 ))
done
echo "Formatted $COUNT queries"
```

### CODE REVIEW

File: assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego [hunk 1]
```diff
@@ -1,91 +1,103 @@
...
-  some i, j
-      splitStr[i] == termOps[j] 
-      indexTerm := indexof(cmd,termOps[j])
-      trimmed := substring(cmd,0,count(cmd) - indexTerm)
-} else = cmd
+	termOps := ["&&", "||", "|", "&", ";"]
+
+	splitStr := split(cmd, " ")
+	some i, j
+	splitStr[i] == termOps[j]
```
Comment (felipe-avelar) [inline assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego:64]: Should remove this indentation?

File: assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego [hunk 2]
```diff
@@ -1,91 +1,103 @@
...
+	indexTerm := indexof(cmd, termOps[j])
+	trimmed := substring(cmd, 0, count(cmd) - indexTerm)
```
Comment (felipe-avelar) [inline assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego:66]: Should remove this indentation?

File: assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego [hunk 3]
```diff
@@ -1,91 +1,103 @@
...
-checkRecommendsFlag(cmd,start,end) {
-  some i
-    i >= start 
-    i <= end
-    hasNoRecommendsFlag(cmd[i])
-} else = false
\ No newline at end of file
+	some i
+	value[i] == "apt-get"
+	idx := i + 1
```
Comment (felipe-avelar) [inline assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego:81]: Should remove this indentation?

File: assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego [hunk 4]
```diff
@@ -1,91 +1,103 @@
...
+} else = -1 {
+	true
+}
+
+getCmdLastIdx(arr, initCmdIdx) = idx {
+	termOps := ["&&", "||", "|", "&", ";"]
+	some i
+	i > initCmdIdx
+	arr[i] == termOps[i]
+	idx := i - 1
```
Comment (felipe-avelar) [inline assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego:91]: Should remove this indentation?

File: assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego [hunk 5]
```diff
@@ -1,91 +1,103 @@
...
+} else = count(arr) - 1 {
+	true
+}
+
+checkRecommendsFlag(cmd, start, end) {
+	some i
+	i >= start
+	i <= end
```
Comment (felipe-avelar) [inline assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego:99]: Should remove this indentation?

**Review by felipe-avelar — COMMENTED** [review]
@rogeriopeixotocx I have a question if we will remove `some` indentations, since it is a loop block...

File: assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego (hunk 2 above)
Comment (rogeriopeixotocx) [reply assets/queries/dockerfile/apt_get_without_no_recommends_flag/query.rego:66]: indentations do not affect some instruction.. I believe that this some instruction could be removed in this case.

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Pull request merged**